server/check-version.js

```diff
@@ -1,46 +1,50 @@
 const { setSetting, setting } = require("./util-server");
 const axios = require("axios");
-const compareVersions = require("compare-versions");
+const { log } = require("../src/util");
 
 exports.version = require("../package.json").version;
 exports.latestVersion = null;
 
+// How much time in ms to wait between update checks
```

Comment:
Add info about current value - 48h

Reply:
I can do it, but I would advise against this. One should not create redundancy with comments, redundancy is likely to become out of sync with time.
Would changing `3600 * 1000 * 48` to `1000 * 60 * 60  * 48` help?
Should I proceed duplicating the current value to the comment? Is this the standard for this project?